datatypes/src/operations/reproject.rs

```diff
@@ -72,7 +72,7 @@ impl Clone for CoordinateProjector {
             from: self.from,
             to: self.to,
             p: Proj::new_known_crs(&self.from.to_string(), &self.to.to_string(), None)
-                .expect("worked before"),
+                .expect("the Proj object should have been constructed"),
```

Comment:
this is not the same information. The same case, combination of SRS, worked before so it must work here.

Reply:
0bc74c57b981a9d1509efdf2b3c5b412f68e404f